src/services/UserService.ts

```diff
@@ -143,12 +133,10 @@ const getInitializedTimeZone = (user: User, timeZone: string): Promise<Initializ
  * initialize preferred units (if not already set)
  */
 const initializeUnits = async (units: string): Promise<InitializedUnits> => {
-  const { unitsAcknowledgedOnMs, preferredWeightSystem, unitsNotificationCreatedMs } =
-    CachedUserService.getUserPreferences();
+  const { unitsAcknowledgedOnMs, preferredWeightSystem } = CachedUserService.getUserPreferences();
 
   const initializedUnits: InitializedUnits = {
     unitsAcknowledgedOnMs,
-    unitsNotificationCreatedMs,
```

Comment:
once we have other feature notifications, we could rethink all this.. we may need a more scalable way of doing this. maybe introduce a `firstShowedOnMs` timestamp since that will scale by user. But keeping them all in a user preference may not be a good idea.. we may need to discuss with BE, later once the feature notifications start rolling in.